src/pages/add/search.vue

```diff
@@ -264,10 +326,11 @@ export default defineComponent({
       }
     );
 
+    const selectedCourses = ref(new Set<Course>());
```

Comment:
一貫性出すんだったらこっちだけどtsだと推論されない？
```suggestion
    const selectedCourses = ref<Course>(new Set());
```